policyengine_us/variables/gov/states/id/tax/income/other_taxes/pbf/id_pbf_liable.py

```diff
@@ -0,0 +1,31 @@
+from policyengine_us.model_api import *
+
+
+class id_pbf_liable(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Liable for the Idaho permanent building fund tax"
+    definition_period = YEAR
+    reference = "https://tax.idaho.gov/wp-content/uploads/forms/EIN00046/EIN00046_11-15-2021.pdf#page=10"
+    defined_for = StateCode.ID
+
+    def formula(tax_unit, period, parameters):
+        # Not required to pay if there is no income tax
+        owes_income_tax_before_credits = (
+            tax_unit("id_income_tax_before_non_refundable_credits", period)
+            <= 0
```

Comment:
and flip line 28, or change the variable name. `owes` indicates that tax > 0
```suggestion
            > 0
```